docs/src/main/sphinx/connector/memsql.rst

```diff
@@ -85,15 +85,23 @@ The connector supports pushdown for a number of operations:
 * :ref:`limit-pushdown`
 * :ref:`topn-pushdown`
 
+.. _singlestore-sql-support:
+
 SQL support
 -----------
 
-The following SQL statements are not yet supported:
-
-* :doc:`/sql/grant`
-* :doc:`/sql/revoke`
-* :doc:`/sql/show-grants`
-* :doc:`/sql/show-roles`
-* :doc:`/sql/show-role-grants`
-
-.. include:: sql-delete-limitation.fragment
\ No newline at end of file
+The connector provides read access and write access to data and metadata in
+a SingleStore database.  In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
```

Comment:
Should this be?
```suggestion
statements, the connector supports the following statements:
```

Reply:
Possibly but again, for consistency/expediency's sake, I'd rather leave as-is since this wording was approved previously across other connectors.